### PR TITLE
Improve styling for reviews, comments, and blockquotes

### LINKED ISSUE #194: Comment blockquotes should be styled better
They need more padding and should be aligned to the edge of the text

image

### PR DESCRIPTION
**Pull request by misolori**
Fixes #194 #195 

**Before**
image

**After**
image

### CODE REVIEW

File: preview-src/index.css [hunk 1]
```diff
@@ -384,12 +406,18 @@ body .comment-form .form-actions button {
 }
 
 .comment-body blockquote {
```
Comment (RMacfarlane) [inline preview-src/index.css]: could you change this to just `blockquote`? https://github.com/xtermjs/xterm.js/pull/1392 has an example of a blockquote being used in somewhere that's not a comment-body

Comment (misolori) [reply preview-src/index.css]: 👍 Opened the scope so it's for all `blockquote` elements

File: preview-src/index.css [hunk 2]
```diff
@@ -384,12 +406,18 @@ body .comment-form .form-actions button {
...
-	margin: 0 7px 0 5px;
-	padding: 0 16px 0 10px;
+	display: flex;
```
[the remark this answers is not in the document]
Comment (misolori) [reply preview-src/index.css]: Thanks! Something weird was happening with the flex property (originally used it to get the paragraph padding to show up). Change it to to a block element and adjusted the padding so it shows up correctly.

[the remark this answers is not in the document]
Comment (misolori) [reply preview-src/index.css]: Should work better with list items:
image

And with longer paragraphs:
image

[the remark this answers is not in the document]
Comment (RMacfarlane) [reply preview-src/index.css]: Awesome! 👍

File: preview-src/index.css [hunk 3]
```diff
@@ -3,11 +3,15 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+ :root {
```
Comment (rebornix) [inline preview-src/index.css]: unnecessary space carried from auto indent on enter.

**Pull request merged**